scripts/src/main/resources/scripts/command/aws

```diff
@@ -63,28 +81,31 @@ function doSetup() {
       AWS_VERSION=$(curl "https://github.com/aws/aws-cli/tags" | awk -F'tags/' '/archive/ { print $2}' | sort -r | head -1 | awk -F'.zip' '{print $1}')
     fi
 
-    doInstall "-" "${DEVON_IDE_HOME}/updates/install/aws" "aws" "${AWS_VERSION}" 
-    if [ "${?}" = 0 ]
+    doInstall "-" "${DEVON_IDE_HOME}/updates/install/aws" "aws" "${AWS_VERSION}"
+    ERR="${?}"
+    if [ "${ERR}" = "0" ]
     then
-      doPackageInstall "${DEVON_IDE_HOME}/updates/install/aws" "${AWS_VERSION}" 
+      doPackageInstall "${DEVON_IDE_HOME}/updates/install/aws" "${AWS_VERSION}"
     fi
+    doRunCommand "rm -rf ${DEVON_IDE_HOME}/updates/install/aws"
 
     if [ "${1}" != "silent" ] && ! doIsQuiet
     then
       doEcho "logon again to test aws with \"aws --version\""
-    fi 
+    fi
   fi
 }
 
 
 # CLI
-case ${1} in 
+case ${1} in
 "help" | "-h")
   echo "Setup or run AWS CLI (command-line interface for Amazon-Web-Services)."
   echo
   echo "Arguments:"
-  echo " setup                          install aws on your machine."
-  echo " «args»                         call aws with the specified arguments. Call aws --help for details or use aws directly as preferred."
+  echo " setup                    install aws on your machine."
+  echo " <<args>>                 call aws with the specified arguments. Call aws --help for details or use aws directly as preferred."
```

Comment:
IMHO @sarahffm changed the indendation of the help output in all commandlets to the same width so that added generic args fit the same indendation width and appear aligned.
Can you check this?

Reply:
Done